src/components/OntologyBrowser/index.tsx

```diff
@@ -54,17 +56,48 @@ const updateSqons = (initialSqon: Sqon, value: string[]) => {
   return [initialSqon];
 };
 
+const generateMsgWhenHasDataButNoSelection = (hasData: boolean, isLoading: boolean) => {
+  if (isLoading || !hasData) {
+    return null;
+  }
+  return 'Select items from the panel on the left in order to add them to your query';
+};
+
+const displaySpinner = () => (
+  <div
```

Comment:
Style shoud go in a css file, plus, I think this should go in a separate component. We should have something like uikit but with antd for pre-baked KF components

Reply:
Good point.